app/src/main/java/com/nemo/androiduitraining/view/util/ViewExt.kt

```diff
@@ -0,0 +1,25 @@
+package com.nemo.androiduitraining.view.util
+
+import android.view.View
+import androidx.databinding.BindingAdapter
+import java.time.Instant
+
+/**
+ * 連打対策付きクリックイベント
+ */
+@BindingAdapter("onClick")
+fun View.setOnClickListenerWithoutDoubleTap(onClickListener: View.OnClickListener?) {
+    setOnClickListener(DoubleTapOnClickListener(onClickListener))
+}
+
+class DoubleTapOnClickListener(val listener: View.OnClickListener?) : View.OnClickListener {
+    private var lastTapAt = 0L // ms
```

Comment:
単位は変数の末尾に含めると読みやすいです。

```suggestion
    private var lastTapAtMs = 0L
```